README.md

```diff
@@ -15,8 +15,8 @@
 * [Advanced search and filtering capabilities](docs/note-filtering.md) including [tags](docs/tags.md), links and mentions
 * [Integration with your favorite editors](docs/editors-integration.md):
     * [Any LSP-compatible editor](docs/editors-integration.md)
-    * [`zk-nvim`](https://github.com/mickael-menu/zk-nvim) for Neovim 0.5+
-    * [`zk-vscode`](https://github.com/mickael-menu/zk-vscode) for Visual Studio Code
+    * [`zk-nvim`](https://github.com/zk-org/zk.nvim) for Neovim 0.5+
```

Comment:
should we rename the link title to `zk.nvim`?

Reply:
also in `editors-integration.md`

Reply:
And we'll need to then change the actual url of the repo after this is changed. 
This change will also break peoples plugin manager conf though. Not a huge issue, as it's easy to remedy on their end, but still. Just wanted to voice it, also as I'm not 100% clear on the benefits of the url change, so I can't appropriately measure the tradeoff.
 
All good for you to make a call on this @mcDevnagh .

Reply:
I don't think you still need to rename from `zk-nvim` to `zk.nvim` now that the repos were transfered. For a fork it would be an issue.

Reply:
I'm in agreement with Mickael. 

The break in plugin managers occurs because most of them save them in directories of the name of the repository, in this case `zk-nvim`, and they don't all delete repositories once downloaded. For [lazy.nvim](https://github.com/folke/lazy.nvim), you specifically need to call [`Lazy clean`](https://github.com/folke/lazy.nvim#-usage) or run clean via the UI. So if a user switches the URL without first cleaning, bad things can happen. However, because the repositories were transferred over, any network requests to https://github.com/mickael-menu/zk redirect to this repo, so end users can just leave in the `https://github.com/mickael-menu/zk` URL or slug (i.e. `mickael-menu/zk`).

New users of the zk-nvim plugin should use the new URL, and existing users can continue using the old.

I'm not even sure that new URLs would break plugin managers; I just know that [none-ls.nvim](https://github.com/nvimtools/none-ls.nvim) thought so. However [null-ls.nvim](https://github.com/jose-elias-alvarez/null-ls.nvim) is archived, not transferred over.

Reply:
I second staying with `zk-nvim` for the repository name.

Reply:
We should stick with `zk-nvim`, in part to avoid confusion with the existing `zk.nvim` plugin.

Reply:
I hadn't even realized that the name was "taken." If we needed to change it, `zk.nvim` should be off the table--we'll have to come up with something else--but it looks like we can stick with `zk-nvim` and we'll never have to switch